restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/AccountsEndpoint.java

```diff
@@ -133,7 +136,8 @@ private Account createFrom(final Sid accountSid, final MultivaluedMap<String, St
         PasswordValidator validator = PasswordValidatorFactory.createDefault();
         if (!validator.isStrongEnough(password))
             throw new PasswordTooWeak();
-        final String authToken = new Md5Hash(password).toString();
+        final List<AuthToken> authToken =new ArrayList<>();
```

Comment:
@arslan70 What if I want to create an account and provide multiple auth tokens? We need to take care of that also.